src/Pim/Component/Catalog/Normalizer/Indexing/Product/ProductValueNormalizer.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace Pim\Component\Catalog\Normalizer\Indexing\Product;
+
+use Pim\Component\Catalog\AttributeTypes;
+use Pim\Component\Catalog\Model\ProductValueInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * Normalize a product value to the indexing format.
+ * This format is based on the standard format but differs from the following:
+ *      - the attribute code is suffixed by its backend type
+ *      - value is indexed by attribute type + backend type...
+ *      - ...then by channel...
+ *      - ... and finally by locale
+ *
+ * If the attribute related to the value is not scopable, here the value will be indexed with the key "<all_channels>".
+ * If the attribute related to the value is not localizable, here the value will be indexed with the key "<all_locales>".
+ *
+ * For instance, imagine we have the following value normalized to the standard format:
+ *  "auto_exposure" => array:1 [
+ *      0 => array:3 [
+ *          "locale" => null
+ *          "scope" => null
+ *          "data" => true
+ *      ]
+ *  ]
+ * where the attribute "a_number_float" (non localizable and non scopable)  is a "pim_catalog_number" attribute.
+ *
+ * Here we'll have:
+ *  "auto_exposure-boolean" => array:1 [
+ *      0 => array:1 [
+ *          "<all_channels>" => array:1 [
+ *              "<all_locales>" => true
+ *          ]
+ *      ]
+ *  ]
+ *
+ * Also, prices are indexed by currency. See the method "normalizePriceCollectionData" below.
+ *
+ * @author    Julien Janvier <jjanvier@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class ProductValueNormalizer implements NormalizerInterface
+{
+    /** @var NormalizerInterface */
+    protected $stdNormalizer;
```

Comment:
Just a question, why not standardNormalizer ? In my head, below when I see stdNormalizer and after stdValues it rings like stdClass values :/.